### ISSUE TITLE
Update `parametrize_sampler` of `test_samplers.py`

### ISSUE THREAD

**Issue by norihitoishida** [optuna/optuna#1993]
## Description
There seem to be two points of improvement. [Link](https://github.com/optuna/optuna/blob/82a313fa890823826ace8cbc2e819afed3b0ca65/tests/samplers_tests/test_samplers.py#L27)

1. Complete `parametrize_sampler` decorator
Now, `parametrize_sampler` doesn't support all sampler (e.g. CmaEsSampler)

```
parametrize_sampler = pytest.mark.parametrize(
    "sampler_class",
    [
        optuna.samplers.RandomSampler,
        lambda: optuna.samplers.TPESampler(n_startup_trials=0),
        lambda: optuna.samplers.TPESampler(n_startup_trials=0, multivariate=True),
        lambda: optuna.integration.SkoptSampler(skopt_kwargs={"n_initial_points": 1}),
        lambda: optuna.integration.PyCmaSampler(),
    ],
)
```

2. Set `n_startup_trials` of `PyCmaSampler()` 
`n_startup_trials` : The random sampling is used instead of the TPE algorithm until the given number of trials finish in the same study.

I'll work on this. If you have any ideas, please let me know :)

**Comment by hvy** [comment]
Let me close this issue as fixed since #2020 which addresses both points has been merged.

**Comment by norihitoishida** [comment]
> Let me close this issue as fixed since #2020 which addresses both points has been merged.

Thanks as always :)
